src/storage/slotted_page.c

```diff
@@ -1514,12 +1514,7 @@ spage_shift_slot_up (PAGE_PTR page_p, SPAGE_HEADER * page_header_p, SPAGE_SLOT *
     }
   else
     {
-      for (; last_slot_p < slot_p; last_slot_p++)
-	{
-	  spage_set_slot (last_slot_p, (last_slot_p + 1)->offset_to_record, (last_slot_p + 1)->record_length,
-			  (last_slot_p + 1)->record_type);
-	}
-      assert (last_slot_p == slot_p);
+      memmove (last_slot_p, last_slot_p + 1, sizeof (SPAGE_SLOT) * (slot_p - last_slot_p));
```

Comment:
```suggestion
      if (last_slot_p < slot_p)
        {
          memmove (last_slot_p, last_slot_p + 1, sizeof (SPAGE_SLOT) * (slot_p - last_slot_p));
        }
```

Reply:
Do you think it needs to do although It will be always true?